docs/cudf/source/cudf_polars/streaming_execution.md

```diff
@@ -0,0 +1,90 @@
+# Experimental streaming and multi-GPU options
+
+As well as in-memory execution, obtained by selecting `engine="gpu"`
+when `collect`ing a query, the GPU engine supports streaming execution
+that partitions the data in the query into chunks. To select streaming
+execution, we need to pass an appropriately configured `GPUEngine`
+object into `collect`.
+
+The streaming executors work best when the inputs to your query come
+from parquet files. That is, start with `scan_parquet`, not existing
+Polars `DataFrame`s or CSV files.
+
+````{note}
+Streaming execution is in an experimental state and not all queries
+will run in streaming mode.
+````
+
+## Single GPU streaming
+
+The simplest case, requiring no additional dependencies, is the
+`synchronous` executor. An appropriate engine is:
+```python
+engine = pl.GPUEngine(
+    executor="streaming",
+    executor_options={"scheduler": "synchronous"},
```

Comment:
Should we note that this is already the `executor="streaming"` default when *nothing* is specified for `executor_options`?

Reply:
I'd say remove this line. If we're planning on adjusting the default to a better scheduler in the future, we might not want people copy-pasting this line around and missing out on those improvements.